.eslintrc.js

```diff
@@ -4,18 +4,8 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  extends: ['eslint:recommended', 'prettier'],
-  plugins: ['prettier'],
+  extends: 'eslint:recommended',
   env: {
     browser: true
-  },
-  rules: {
```

Comment:
This was wreaking havoc on the test suite.  Lmk if this is important to everybody and I can add it back!

Reply:
Hey, we use this to lint along with prettier.
Please re-implement this and get the test suite passing.

If you need any help you can look here:
https://prettier.io/docs/en/eslint.html